dev-packages/application-package/src/generator/webpack-generator.ts

```diff
@@ -70,6 +70,19 @@ module.exports = {
                 loader: 'ignore-loader'
             },
             {
+                /* Get rid of :
```

Comment:
Get => Gets?

Reply:
I am not sure what verb tense it's called, but I mean to use some kind of "impersonal present", like in a recipe.  Like I would say:

```
// Add x and y.
var foo = x + y;
// Clamp to the threshold.
foo = min (foo, threshold);
```

Reply:
Actually it's probably the imperative tense in that case.  You intimidate the computer into doing what you want.